locales/en/common.json

```diff
@@ -4,7 +4,10 @@
         "subtitle": "We respect your inbox. No spam, promise ✌️",
         "nameInput": "Your name",
         "emailInput": "Your email",
-        "subscribeBtn": "Subscribe"
+        "subscribeBtn": "Subscribe",
+        "successTitle": "Thank you for subscribing!",
+        "errorTitle": "Something went wrong",
+        "errorSubtitle": "Please try again later."
```

Comment:
all looks good to me except of the error messages

in many cases if these will show up, we probably messed something on our side and waiting will not solve anything. Better if people let us know.

so instead of "something ain't right, wait" we should have an error message saying "Subscription failed, please let us know about it but submitting a bug https://github.com/asyncapi/website/issues/new?template=bug.md"

Reply:
Done 👍 . Updated in both English and German Languages.